page-searchbox_multi.php

```diff
@@ -1,11 +1,11 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<!DOCTYPE html>

```

Comment:
I thought maybe the github syntax errors would go away if I updated the DOCTYPE to regular ol' HTML 5, but no dice. This file passes the W3C HTML validator. 
If I run it through a _php_ validator, I have to start with `<?php ?>` in order to get it to  pass, but it does.
I'm hesitant to merge code with "syntax errors", though.  @moseshll What do you think?

Reply:
Merge it. I trust your eyeballs more than I trust GitHub's algos.